docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -189,7 +216,7 @@ You can delete a user from your Temporal Cloud Account by using either Web UI or
 
 :::info
 
-To delete a user, a user must have the Global Admin account-level [Role](#account-level-roles).
+To delete a user, a user must have the Account Owner or Global Admin account-level [Role](#account-level-roles).
```

Comment:
To delete a user who is not an Account Owner, the person performing the deletion must have either Account Owner or Global Admin privileges.